tests/unit/phase/test_organisation.py

```diff
@@ -0,0 +1,40 @@
+from digital_land.organisation import Organisation
+from digital_land.phase.organisation import OrganisationPhase
+from digital_land.log import IssueLog
+
+
+def test_process_with_valid_organisation():
+    organisation = Organisation(
```

Comment:
this should be mocked rather than using actual data. Especially for a unit test as in theory they assume that there's no access to a file system. Consider something like 

    def test_process_with_valid_organisation(mocker):

        mocker.patch(
            "digital_land.phase.organisation.Organisation.organisation.lookup",
            return_value='valid_org'
        )
    
This means that the test doesn't have to load data in from the file system and doesn't rely on dependencies outside of your code.

If you wanted to write a test including the organisation class it should be an integration test and I would advice creating your own organisation.csv in temporary location using tmp_path

Reply:
I was under the impression we test the entire workflow of organisation, but you are right, this is a unit test so should mock the main organisation.
Done the changes now